templates/navigation-menu.tpl

```diff
@@ -25,7 +25,12 @@
         {/if}
         {if $nav__canBan || $nav__canEmailMgmt || $nav__canWelcomeMgmt || $nav__canSiteNoticeMgmt || $nav__canUserMgmt || $nav__canJobQueue || $nav__canFlaggedComments || $nav__canQueueMgmt || $nav__canDomainMgmt || $nav__canErrorLog}
             <li class="nav-item dropdown">
-                <a href="#" class="nav-link dropdown-toggle" aria-haspopup="true" aria-expanded="false" data-toggle="dropdown"><i class="fas fa-wrench"></i>&nbsp;Admin</a>
+                <a href="#" class="nav-link dropdown-toggle" aria-haspopup="true" aria-expanded="false" data-toggle="dropdown">
+                    <i class="fas fa-wrench"></i>&nbsp;Admin
+                    {if $nav__numAdmin > 0}
+                        <div class="badge badge-danger">{$nav__numAdmin}</div>
+                    {/if}
+                </a>
```

Comment:
Just thinking - how does this work on reduced-width screens? There's newly-added whitespace here (which isn't covered with nonbreaking spaces), so I'm not sure if the browser will try to wrap the text onto multiple lines.

I'll double-check this myself in a local test environment (unless you've already checked this?) so don't worry about making changes just yet (this is more of a note for myself)

Reply:
Never mind; other things break on the navbar before this:
![Screenshot from 2022-07-29 23-26-01](https://user-images.githubusercontent.com/722710/181853339-d3c75a59-2edb-4760-9792-4ce2d2f0f5b9.png)

(and that's a contrived worst-case example)